site/content/how-to/data-plane-configuration.md

```diff
@@ -112,3 +112,48 @@ Status:
 ```
 
 If everything is valid, the `ResolvedRefs` condition should be `True`. Otherwise, you will see an `InvalidParameters` condition in the status.
+
+## Configure the Data Plane Log Level
+
+You can use the `NginxProxy` resource to dynamically configure the Data Plane Log Level.
+
+The following command creates a basic `NginxProxy` configuration that sets the log level to `warn` instead of the default value of `info`:
+
+```yaml
+kubectl apply -f - <<EOF
+apiVersion: gateway.nginx.org/v1alpha1
+kind: NginxProxy
+metadata:
+  name: ngf-proxy-config
+spec:
+  logging:
+    errorlevel: warn
+EOF
+```
+
+After attaching the NginxProxy to the GatewayClass, the log level of the data plane will be updated to `warn`.
+
+To view the full list of supported log levels, see the `NginxProxy spec` in the [API reference]({{< relref "reference/api.md" >}}).
+
+{{< note >}}For `debug` logging to work, NGINX needs to be built with `--with-debug` or "in debug mode". NGINX Gateway Fabric can easily
```

Comment:
So we're not going to explain (or link to) building nginx and the container for debug mode?

Reply:
Does the following section --_Run NGINX Gateway Fabric with NGINX in debug mode_ -- not suffice? 

The only thing I could see it not working for is for previous versions of NGF, which I could add a note if needed.

Reply:
Well we have to build the nginx debug binary and then rebuild the nginx container. A user won't know how to do that. @kate-osborn how do you think we should convey this? Does it make sense to have all of this info here?

Reply:
So we found out that the nginx container already contains the nginx debug binary. So all we have to do is change the container command to use the nginx debug binary.

Reply:
Oh I didn't realize that. Does the nginx-plus container contain both? Is there an nginx-plus-debug binary?

Reply:
I'm pretty sure @bjee19 tested with nginx plus. I believe it also has the `nginx-debug` binary.

Reply:
Nice!